src/Symfony/composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "open-telemetry/contrib-symfony-instrumentation-bundle",
+    "name": "open-telemetry/contrib-sdk-bundle",
```

Comment:
I'm not entirely sure what packagist will do with the name change, but it's still alpha so I'm ok with trying it out and see what happens.

Reply:
Let's revert it, or at least keep "symfony" in the name to help with discoverability

Reply:
Has `open-telemetry/contrib-symfony-instrumentation-bundle` been submitted to Packagist yet? I don't seem to be able to find it there, which might go someway to explain the issues I was having

Reply:
Yes. It looks like it's this one: https://packagist.org/packages/open-telemetry/contrib-sdk-bundle (it uses the package name from composer.json)

Reply:
ah, so should it be submitted to Packagist under the new name?

Reply:
This is a good time to change it. I think `open-telelemetry/symfony-sdk-bundle` is short and descriptive. WDYT?

Reply:
That works... it does what it says on the tin!